docs/editing-components.md

```diff
@@ -244,6 +244,7 @@ When comparing against multiple `values`, the comparison will be true if at leas
 * `>=`
 * `typeof`
 * `regex`
+* `length-equals`
```

Comment:
Yep, I'd change the name to `length-equals` (since "length" is used for other things instead of "size" and so the configs read better). If you're already here it might be nice to also add `length-greater-than` and `length-less-than` since I could easily see them being used.

Plus the docs should say that it compares the length of strings and arrays